app/org/sagebionetworks/bridge/models/schedules/SchedulerFactory.java

```diff
@@ -0,0 +1,18 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import org.sagebionetworks.bridge.validators.ScheduleValidator;
+import org.sagebionetworks.bridge.validators.Validate;
+
+public class SchedulerFactory {
```

Comment:
Respectfully disagree. Looking at the big picture is an important part of the code review process.

Also, unless I'm looking at the wrong page, I'm not seeing the design details in https://sagebionetworks.jira.com/wiki/display/BRIDGE/Schedule